decidim-admin/app/controllers/decidim/admin/features/base_controller.rb

```diff
@@ -8,7 +8,6 @@ module Features
       class BaseController < Admin::ApplicationController
         skip_authorize_resource
         include Concerns::ParticipatoryProcessAdmin
-        include NeedsParticipatoryProcess
```

Comment:
To build on that a bit, the next PR from me should be to make features polymorphic and thus removing this dependency you're talking about.